src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -798,4 +798,39 @@
   <data name="Profile_DeleteConfirmationMessage.Text" xml:space="preserve">
     <value>Are you sure you want to delete this profile?</value>
   </data>
+  <data name="Globals_TabSwitcherMode.Header" xml:space="preserve">

```

Comment:
@cinnamon-msft since you just went through all the localized text. Mind taking a look here?